### ISSUE TITLE
Is there a more detailed sklearn api?

### ISSUE THREAD

**Issue by Skyexu** [aksnzhy/xlearn#291]
I can't find some function like `fm.disableNorm()` or `.cv()` in sklearn api,  
does  it work ,when set fold below?

`fm_regressor = xl.FMModel(task='reg', metric='rmse', epoch=500, k=4, stop_window=10, lr=0.0001, reg_lambda=0.01,fold=5)`

thank u

**Comment by ankane** [comment]
Hey @Skyexu, there's a parameter in the `fit` method for `disableNorm` and other options.

```python
model.fit(X, y, is_instance_norm=False)
```

https://github.com/aksnzhy/xlearn/blob/e03f08ab22779ce13d9b08ac895ef5ba18fb16ef/python-package/xlearn/_sklearn.py#L187-L189

There doesn't appear to be a `cv` method.

**Comment by Skyexu** [comment]
@ankane ok  thank u 😁
